docs/about/release-notes.md

```diff
@@ -1,5 +1,35 @@
 # Release Notes
 
+
+## ShapeWorks 6.2.1 - 2022-01-07
+
+### What is new?
+
+#### Studio
```

Comment:
For consistency, I would use the same four sub-categories that we have been using in previous release notes. ShapeWorks Front-end, ShapeWorks Back-end, Deep Learning, and User Support.

Reply:
so this list should belong to front-end sub-category.